From 65b7123e7017dd2773151c5cbf0e65681724be6a Mon Sep 17 00:00:00 2001
Message-Id: <patch-5.6-65b7123e701-20210615T161330Z-avarab@gmail.com>
In-Reply-To: <cover-0.6-00000000000-20210615T161330Z-avarab@gmail.com>
References: <cover-0.6-00000000000-20210615T161330Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 15 Jun 2021 18:02:31 +0200
Subject: [PATCH 5/6] doc: replace "alice" and "bob" with "jdoe" and "msmith"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the "Alice" and "Bob" generic example users to jdoe@example.com
and msmith@example.com.

The former is widely used in RFC 5322 as an example E-Mail address,
the latter is not, but "Mary Smith <mary@example.com>". It has been
claimed that any reference to people's gender in our documentation is
distracting to some readers[1]. In this case it's easy enough to tweak
the example in such a way that the reader can insert their own
stand-in for "M.".

1. https://lore.kernel.org/git/pull.975.v3.git.1623766273.gitgitgadget@gmail.com/

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/git-credential.txt         |  2 +-
 Documentation/git-imap-send.txt          |  4 ++--
 Documentation/git-interpret-trailers.txt | 22 +++++++++++-----------
 3 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/Documentation/git-credential.txt b/Documentation/git-credential.txt
index 206e3c5f407..86b4c32affa 100644
--- a/Documentation/git-credential.txt
+++ b/Documentation/git-credential.txt
@@ -69,7 +69,7 @@ information it has):
 
 	protocol=https
 	host=example.com
-	username=bob
+	username=jdoe
 	password=secr3t
 +
 In most cases, this means the attributes given in the input will be
diff --git a/Documentation/git-imap-send.txt b/Documentation/git-imap-send.txt
index 63cf498ce9f..902f0e8ed34 100644
--- a/Documentation/git-imap-send.txt
+++ b/Documentation/git-imap-send.txt
@@ -72,7 +72,7 @@ Using direct mode:
 [imap]
     folder = "INBOX.Drafts"
     host = imap://imap.example.com
-    user = bob
+    user = jdoe
     pass = p4ssw0rd
 .........................
 
@@ -82,7 +82,7 @@ Using direct mode with SSL:
 [imap]
     folder = "INBOX.Drafts"
     host = imaps://imap.example.com
-    user = bob
+    user = jdoe
     pass = p4ssw0rd
     port = 123
     ; sslVerify = false
diff --git a/Documentation/git-interpret-trailers.txt b/Documentation/git-interpret-trailers.txt
index 956a01d184f..17a19960ba3 100644
--- a/Documentation/git-interpret-trailers.txt
+++ b/Documentation/git-interpret-trailers.txt
@@ -277,13 +277,13 @@ $ cat msg.txt
 subject
 
 message
-$ cat msg.txt | git interpret-trailers --trailer 'sign: Alice <alice@example.com>' --trailer 'sign: Bob <bob@example.com>'
+$ cat msg.txt | git interpret-trailers --trailer 'sign: J. Doe <jdoe@example.com>' --trailer 'sign: M. Smith <msmith@example.com>'
 subject
 
 message
 
-Signed-off-by: Alice <alice@example.com>
-Signed-off-by: Bob <bob@example.com>
+Signed-off-by: J. Doe <jdoe@example.com>
+Signed-off-by: M. Smith <msmith@example.com>
 ------------
 
 * Use the `--in-place` option to edit a message file in place:
@@ -294,15 +294,15 @@ subject
 
 message
 
-Signed-off-by: Bob <bob@example.com>
-$ git interpret-trailers --trailer 'Acked-by: Alice <alice@example.com>' --in-place msg.txt
+Signed-off-by: M. Smith <msmith@example.com>
+$ git interpret-trailers --trailer 'Acked-by: J. Doe <jdoe@example.com>' --in-place msg.txt
 $ cat msg.txt
 subject
 
 message
 
-Signed-off-by: Bob <bob@example.com>
-Acked-by: Alice <alice@example.com>
+Signed-off-by: M. Smith <msmith@example.com>
+Acked-by: J. Doe <jdoe@example.com>
 ------------
 
 * Extract the last commit as a patch, and add a 'Cc' and a
@@ -311,7 +311,7 @@ Acked-by: Alice <alice@example.com>
 ------------
 $ git format-patch -1
 0001-foo.patch
-$ git interpret-trailers --trailer 'Cc: Alice <alice@example.com>' --trailer 'Reviewed-by: Bob <bob@example.com>' 0001-foo.patch >0001-bar.patch
+$ git interpret-trailers --trailer 'Cc: J. Doe <jdoe@example.com>' --trailer 'Reviewed-by: M. Smith <msmith@example.com>' 0001-foo.patch >0001-bar.patch
 ------------
 
 * Configure a 'sign' trailer with a command to automatically add a
@@ -326,12 +326,12 @@ $ git config trailer.sign.command 'echo "$(git config user.name) <$(git config u
 $ git interpret-trailers <<EOF
 > EOF
 
-Signed-off-by: Bob <bob@example.com>
+Signed-off-by: M. Smith <msmith@example.com>
 $ git interpret-trailers <<EOF
-> Signed-off-by: Alice <alice@example.com>
+> Signed-off-by: J. Doe <jdoe@example.com>
 > EOF
 
-Signed-off-by: Alice <alice@example.com>
+Signed-off-by: J. Doe <jdoe@example.com>
 ------------
 
 * Configure a 'fix' trailer with a key that contains a '#' and no
-- 
2.32.0.555.g0268d380f7b

